showpass/Showpass-E2E-tests
# TA-99 Verify the 'Create Account' invalid inputs and error handling

**Description:**
This user story will be making sure that the invalid inputs and errors are being properly handled by the ‘Create Account’ modal window. 

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-99

**How to test:**
Run the following spec and ensure it passes: cypress/e2e/registration/registration-error-handling.cy.js

**Test execution results:**

<img width="725" alt="TA-99-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/2b3051e3-1231-4de3-afe2-e64b9c0b10fd">
